### PR TITLE
First pass to replace iconomist and favicon with responsive_favicons

### PR DESCRIPTION
**Pull request by typhonius**
This addresses #31 - it's a super rough first pass that we can build on so please be gentle :kissing:

### CODE REVIEW

File: build/tests/behat/features/disabled-modules.feature [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Feature: Disabled modules
+
+  Ensure deprecated modules are not available for install
+
+  @api @disabled-modules
+  Scenario: View the module page
+    Given I am logged in as a user named "sean with the "administrator" role that doesn't force password change
```
Comment (seanhamlin) [inline build/tests/behat/features/disabled-modules.feature]: quotation issues here?

**Comment by fiasco** [comment]
ping @typhonius tests are failing, can you please fix them?

**Comment by typhonius** [comment]
`govcms_tweaks` doesn't get enabled for non-ACSF installs so this should get put into the govcms profile I think.

**Comment by typhonius** [comment]
@fiasco this isn't failing because of my patch anymore - it's a robots.txt test.

**Comment by fiasco** [comment]
@typhonius can we get a rebase for this?

**Comment by fiasco** [comment]
This pull request needs to be re-submitted against the `7.x-2.x` branch as the master branch has been removed per #226

Apologies for the inconvenience.

**Pull request closed without merging**